test.sh

```diff
@@ -16,3 +16,6 @@ fgt golint ./...
 go test ./...
 go list -f '{{if len .TestGoFiles}}"go test -coverprofile={{.Dir}}/.coverprofile {{.ImportPath}}"{{end}}' ./... | xargs -i sh -c {}
 gover . coverprofile.txt
+go get -d ./...
```

Comment:
Aha, I see. It's probably worth diverging from the BUILDING.md file in Travis, because (a) the get is redundant, and (b) it looks like this go get is breaking the Travis build because go disagrees with Travis about the name of the repo (.../cfssl.git vs .../cfssl).